packages/fxa-auth-server/package.json

```diff
@@ -47,7 +47,7 @@
     "subscription-reminders": "CONFIG_FILES='config/secrets.json' node -r esbuild-register ./scripts/subscription-reminders.ts",
     "audit-orphaned-stripe-accounts": "CONFIG_FILES='config/secrets.json' node -r esbuild-register ./scripts/audit-orphaned-customers.ts",
     "remove-unverified-accounts": "CONFIG_FILES='config/secrets.json' node -r esbuild-register ./scripts/remove-unverified-accounts.ts",
-    "storybook": "NODE_OPTIONS=--openssl-legacy-provider storybook dev -p 6010 --no-version-updates -s ./",
+    "storybook": "NODE_OPTIONS=--openssl-legacy-provider storybook dev -p 6010 --no-version-updates ./",
```

Comment:
Good catch on a deprecated flag. Should we configure more specific static dirs? Definitely not a show stopper, just something for another time.

Reply:
We have staticDirs also configured in `.storybook/main.js`, so launching the storybook was resulting in an error (needed to configure static dirs in only one spot). So I think we're covered here and no follow-up required :)